.github/workflows/checks.yml

```diff
@@ -134,6 +134,7 @@ jobs:
         if: steps.cache-contracts.outputs.cache-hit != 'true'
         run: |
           pip install --upgrade setuptools
+          pip install sympy==1.11.1
```

Comment:
Note: For some reason pip installs an incorrect `sympy` version that is not compatible with the [required version](https://github.com/starkware-libs/cairo-lang/blob/0e4dab8a6065d80d1c726394f5d9d23cb451706a/scripts/requirements.txt#L58) for `cairo-lang`. As a current workaround it is installed manually. If we decide to remove tests for Cairo 0, this job will also be removed.